src/jsonrpc.ts

```diff
@@ -0,0 +1,95 @@
+import {Buffer} from 'buffer';
+import {NEAR} from 'near-units';
+import {JsonRpcProvider, ContractCodeView, AccountView, NearProtocolConfig, AccountBalance, CodeResult, BlockId, Finality, ViewStateResult} from './types';
+import {Records} from './contract-state';
+
+export class JSONRpc extends JsonRpcProvider {
```

Comment:
Could you add TSDoc comments to this class to explain why it's worth using this instead of `JsonRpcProvider` from NAJ? Based on the name it's unclear why I'd prefer one over the other.

Reply:
I'll update it.  It just provides more functionality than the super class.  So perhaps I'll just alias `JsonRpcProvider` at the import and then rename the suclass as the super.  Eventually this should make it's way back into NAJ.